src/methods/runSync.js

```diff
@@ -23,6 +23,8 @@ const syncRunner = async function () {
 		if (lockRelease) {
 			lockRelease()
 		}
+		
```

Comment:
Hmm.. This SHOULD be fine.  I don't think we have been expecting a response from this method previously.

Why do we need to do this change, though?